oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/DocumentStoreIndexerBase.java

```diff
@@ -251,7 +238,7 @@ public void reindex() throws CommitFailedException, IOException {
 
         closer.register(indexer);
 
-        FlatFileStore flatFileStore = buildFlatFileStore(checkpointedState, indexer);
+        FlatFileStore flatFileStore = buildFlatFileStore(checkpointedState, indexer, indexer::shouldInclude, null);
```

Comment:
just a nit pick.
this seems redundant since we are already using the same predicate in line 161 too in case the pathPredicate passed to that method is null.
I think either one of these could be removed (I don't have a strong opinion around this though.)